### ISSUE TITLE
accuracy and loss not changing while training.

### ISSUE THREAD

**Issue by minhajfalaki** [tflearn/tflearn#955]
HI guys,
I am getting this result in my training.

| Momentum | epoch: 001 | loss: 0.32643 - acc: 0.4493 -- iter: 96480/96700

Training Step: 3016  | total loss: [1m[32m0.33663[0m[0m | time: 10705.833s

| Momentum | epoch: 001 | loss: 0.33663 - acc: 0.4450 -- iter: 96512/96700

Training Step: 3017  | total loss: [1m[32m0.32481[0m[0m | time: 10709.372s

| Momentum | epoch: 001 | loss: 0.32481 - acc: 0.4443 -- iter: 96544/96700

Training Step: 3018  | total loss: [1m[32m0.33106[0m[0m | time: 10712.895s

| Momentum | epoch: 001 | loss: 0.33106 - acc: 0.4498 -- iter: 96576/96700

 Momentum | epoch: 001 | loss: 0.32079 - acc: 0.4486 -- iter: 96608/96700

Training Step: 3020  | total loss: [1m[32m0.31725[0m[0m | time: 10719.973s

| Momentum | epoch: 001 | loss: 0.31725 - acc: 0.4475 -- iter: 96640/96700

Training Step: 3021  | total loss: [1m[32m0.31622[0m[0m | time: 10723.480s

| Momentum | epoch: 001 | loss: 0.31622 - acc: 0.4496 -- iter: 96672/96700
2017-11-08 03:04:42,096 INFO - Training Step: 3022  | total loss: [1m[32m0.32049[0m[0m | time: 10729.539s

| Momentum | epoch: 001 | loss: 0.32049 - acc: 0.4453 | val_loss: 0.27520 - val_acc: 0.1200 -- iter: 96700/96700

I am trying to train my image datasets(from a camera fitted on a remote control car) with labels as values [-1,1 and 0] corresponding to left,right and straight.

I am using alexnet.py. I am new to CNN why i am getting this problem and how to rectify this?

**Comment by minhajfalaki** [comment]
hi guys, 
in any case if anybody is having same problem with[ alexnet.py](https://github.com/tflearn/tflearn/blob/master/examples/images/alexnet.py) as i posted before.
try some changes as given below. i got 87% accuracy, validation accuracy around 93, loss=0.12 and validation loss=0.12. This may be helpful.

I then called it as a function alexnet (as you can see)for training. the arguments were width, height (of images) and learning rate(which i selected tobe 0.0001) .




import tflearn
from tflearn.layers.conv import conv_2d, max_pool_2d
from tflearn.layers.core import input_data, dropout, fully_connected
from tflearn.layers.estimator import regression
from tflearn.layers.normalization import local_response_normalization

def alexnet(width, height, lr):
    
    network = input_data(shape=[None, width, height,1], name='input')
    network = conv_2d(network, 96, 11, strides=6, activation='relu',regularizer='L2',    weight_decay=0.0005,bias_init='uniform', trainable=True, restore=True)

    network = max_pool_2d(network, 7, strides=2)
    network = local_response_normalization(network)
    network = conv_2d(network, 256, 5, activation='relu',regularizer='L2', weight_decay=0.0005,bias_init='uniform', trainable=True, restore=True)

    network = max_pool_2d(network, 3, strides=2)
    network = local_response_normalization(network)
    network = conv_2d(network, 384, 3, activation='relu')
    network = conv_2d(network, 384, 3, activation='relu')
    network = conv_2d(network, 256, 3, activation='relu')
    network = max_pool_2d(network, 3, strides=2)
    network = local_response_normalization(network)
    network = fully_connected(network, 4096, activation='relu')
    network = dropout(network, 0.5)
    network = fully_connected(network, 4096, activation='relu')
    network = dropout(network, 0.7)
    network = fully_connected(network, 2, activation='softmax')
    network = regression(network, optimizer='momentum',
                         loss='categorical_crossentropy',
                         learning_rate=lr, name='targets')

    model = tflearn.DNN(network, checkpoint_path='model_alexnet',
                        max_checkpoints=1, tensorboard_verbose=2, tensorboard_dir='log')
    
    return model
